hi-ml-multimodal/src/health_multimodal/image/utils.py

```diff
@@ -0,0 +1,53 @@
+#  -------------------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+#  -------------------------------------------------------------------------------------------
+
+import tempfile
+from pathlib import Path
+
+from torchvision.datasets.utils import download_url
+
+from .. import REPO_URL
+from .model import ImageModel
+from .inference_engine import ImageInferenceEngine
+from .data.transforms import create_chest_xray_transform_for_inference
+
+IMAGE_WEIGHTS_NAME = ""  # TODO
+IMAGE_WEIGHTS_URL = f"{REPO_URL}/raw/main/{IMAGE_WEIGHTS_NAME}"
+IMAGE_WEIGHTS_MD5 = ""  # TODO
+
+
+def _download_image_model_weights() -> Path:
+    """Download image model weights from Hugging Face."""
+    root_dir = tempfile.gettempdir()
```

Comment:
Shouldn't we place them in a dedicated directory instead of downloading into different place in each execution? I think that's how other libraries handle it, no?

Reply:
`tempfile.gettempdir()` is deterministic. For example, it's `/tmp` on Linux. I think you're thinking of `tempfile.mkdtemp`.

Another option, slightly more intrusive and maybe not necessary here, is using a cache dir in home. For example, `~/.cache/biovil`.